dac_calibration_server.py

```diff
@@ -102,15 +99,11 @@ def loadServerSettings(self):
             'bandwidthIQ': 0.4, #original default: 0.4
             'bandwidthZ': 0.13, #original default: 0.13
             'maxfreqZ': 0.45, #optimal parameter: 10% below Nyquist frequency of dac, 0.45
-            'maxvalueZ': 5.0, #optimal parameter: 5.0, from the jitter in 1/H fourier amplitudes
-            'dither': True #enable dithering on Z.
+            'maxvalueZ': 5.0 #optimal parameter: 5.0, from the jitter in 1/H fourier amplitudes
         }
         for key in keys.SERVERSETTINGVALUES:
             default = defaults.get(key, None)
-            keyval = yield reg.get(key, False, default)
-            if not isinstance(keyval, bool):
-                #keyval is a number, in labrad units
-                keyval = keyval
+            keyval = default
             print key, ':', keyval
             d[key] = keyval
```

Comment:
I just realized that this whole thing is kinda bogus; why not get rid of the `keys.SERVERSETTINGVALUES` array and build the `serverSettings` dict directly:
```python
self.serverSettings = {
    'deconvIQ': True,
    ...
    'dither': False
}
```